mscore/instrwidget.cpp

```diff
@@ -99,7 +99,35 @@ void ScoreOrderListModel::rebuildData()
 
 void ScoreOrderListModel::setCustomisedOrder(ScoreOrder* order)
       {
-      customisedOrder = order->isCustomised() ? order : nullptr;
+      customisedOrder = (order && order->isCustomised()) ? order : nullptr;
```

Comment:
It rather should be "Cistomizded", with a "z", US spelling

Reply:
Ah! I'll never get used to this US spelling :wink:. It might occur at several places. 
Better to change it?

Reply:
Yes, please

Reply:
Okay. Will affect PR #7194 too.

Reply:
Done, both PR's.